website/addons/mendeley/static/node-cfg.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
 var CitationsNodeConfig = require('js/citationsNodeConfig').CitationsNodeConfig;
-require('./node-cfg.css');
```

Comment:
@sloria Yes, I already deleted it in this PR. I searched the keyword "node-cfg.css" and found it is only contained in the code for mendeley. Since it is an empty file, it doesn't apply any style to the page.